packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -137,6 +147,24 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
         commands.registerCommand({ id: 'workbench.action.files.openFolder' }, {
             execute: () => commands.executeCommand(WorkspaceCommands.OPEN_FOLDER.id)
         });
+        commands.registerCommand({ id: 'workbench.action.gotoLine' }, {
+            execute: () => commands.executeCommand('editor.action.gotoLine')
+        });
+        commands.registerCommand({ id: 'actions.find' }, {
+            execute: () => commands.executeCommand(CommonCommands.FIND.id)
+        });
+        commands.registerCommand({ id: 'undo' }, {
+            execute: () => commands.executeCommand(CommonCommands.UNDO.id)
+        });
+        commands.registerCommand({ id: 'workbench.action.closeAllEditors' }, {
```

Comment:
it is already implemented, we cannot always just reuse command from Theia, such concepts as areas, editors, widgets are not matching. So we should implement them to align properly with VS Code. `Close All Tabs` closes all tabs, not editors in the sense of VS Code.

Reply:
fixed